bemani/backend/core/eacoin.py

```diff
@@ -131,7 +131,7 @@ def make_resp(status: int, balance: int) -> Node:
                 return make_resp(2, 0)
 
             userid = self.data.local.user.from_session(session)
-            if session is None:
```

Comment:
We have already checked nullable for session on L128. So I think we should check nullable for userid now.

Reply:
Ooh nice catch!